docs/application/native/guides/app-management/ui-app/efl-ui-app.md

```diff
@@ -214,12 +214,12 @@ Since Tizen 2.4, an application is not allowed to run in the background except w
 
```

Comment:
On line 205, `[Application Controls](app-controls.md)` is broken. `[Application Controls](../app-controls.md)` looks correct.

Reply:
done.